AppCypher/EventsManager
# #153098355 Deploying Server Code on Heroku

#### What does the PR do?
Deploying the server code to work on heroku

#### Description of task to be completed 
- Add heroku procfiles
- Run server code successfully on heroku

#### How should this be manually tested?
- Clone the project repository 
  
  ```$ git clone https://github.com/AppCypher/EventsManager```

- Checkout into the proper branch 

  ```$ git checkout ch-heroku-deploy-153098355```

- This project requires Nodejs and it is expected to run in Nodejs version 6 or higher

- Install project dependencies by running

  ```$ npm install```

- There project bundles task runners that make trying out parts of the project easy to use
- Below is a list of some of the available task runners
    - `server` to start the server 

    - `test:api` to run the api test suite

    - `build` to start webpack build on all server, frontend and test suite code

    - `build:server` to start webpack build on server code
    
    - `build:client:dev` to start webpack build on client code under a dev environment variable
    
    - `build:client:prod` to start webpack build on client code under a production environment variable

- To run the server or test suite, the project code needs to be built 

  ```$ npm run build```


#### Relevant pivotal tracker stories
153098355
